app/graphql/types/credit_notes/estimate.rb

```diff
@@ -13,6 +13,7 @@ class Estimate < Types::BaseObject
       field :max_refundable_amount_cents, GraphQL::Types::BigInt, method: :refund_amount_cents, null: false
       field :sub_total_excluding_taxes_amount_cents, GraphQL::Types::BigInt, null: false
       field :taxes_amount_cents, GraphQL::Types::BigInt, null: false
+      field :precise_taxes_amount_cents, GraphQL::Types::Float, null: false
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `precise_taxes_amount_cents` should appear before `taxes_amount_cents`.